### DISCUSSION TITLE
Watermark on rtsp

### DISCUSSION THREAD

**Discussion by ArclonT** [roleoroleo/yi-hack-Allwinner-v2#169] [Q&A] [answered]
Hi,
would like to ask if it is possible remove Yi watermark from RTSP stream? (The setting for snapshot watermark is off).

**Comment by roleoroleo** [comment, marked as answer]
It's not possible, read this issue:
https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/19

**Reply by ArclonT** [reply]
Thx a lot. 
I went through it and obviously I do not get it. Can You plz show me more about it?
It's a pity, cause we would like to use it for some streaming of local sport event and with watermark does not look well..

**Reply by roleoroleo** [reply]
Sorry, I fixed the link.
